packages/manager/apps/sap-features-hub/src/pages/installation/initialStep/InstallationInitialStep.page.tsx

```diff
@@ -51,73 +50,69 @@ export default function InstallationInitialStep() {
   );
 
   useEffect(() => {
-    setIsFormValid(clusterName && !isError);
+    setIsStepValid(clusterName && !isError);
```

Comment:
When setIsStepValid is never used outside this useEffect you can just use `useMemo` to define `isStepValid`